src/uwtools/drivers/forecast.py

```diff
@@ -121,15 +129,92 @@ def requirements(self) -> None:
         ???
         """
 
-    def resources(self) -> None:
+    def resources(self, platform: dict) -> Mapping:
         """
-        ???
+        Set up the formatting for the scheduler batch script.
         """
-
-    def run(self):
+        # Add required fields to platform.
+        # Currently supporting only slurm and none.
+        slurmqueues = ["hera", "jet", "orion", "stampede"]
```

Comment:
None of this information belongs in the driver. We haven't talked about this explicitly, but we will need some utility that transforms platform information into Driver-usable information without any additional processing here.

Reply:
OK, I was wondering about that. We'll have to talk about whether to include that as part of this PR or I'll have to put it in the YAML/schema to get the proper run command